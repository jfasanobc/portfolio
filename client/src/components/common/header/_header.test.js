import React from 'react'
import renderer, { create } from "react-test-renderer";
import HeaderBlock from './header.js'

describe("Render HeaderBlock", () => {
  test("Is it rendered and matching", () => {
    const headBlock = create(<HeaderBlock />)
    expect(headBlock.toJSON()).toMatchSnapshot()
  })
})