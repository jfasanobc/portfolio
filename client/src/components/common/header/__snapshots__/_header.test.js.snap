// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HeaderBlock Is it rendered and matching 1`] = `
<header
  className="header row"
>
  <a
    href="/"
  >
    <img
      className="logo"
      src="./assets/img/logo.png"
    />
  </a>
  <p>
    Suh Dude!
    <span>
      @JohnFasano
    </span>
  </p>
</header>
`;
